stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1589,18 +1606,54 @@ internal extension _ExecutionContext {
 @inlinable
 public func withDevice<R>(_ kind: DeviceKind,
                           perform body: () throws -> R) rethrows -> R {
-  _ExecutionContext.global.pushDeviceScope(kind)
+  // internalConsistencyCheck(false, "pushing device scope \(kind).")
+  _ExecutionContext.global.pushDeviceScope(kind, 0)
   let result = try body()
   _ExecutionContext.global.popDeviceScope()
   return result
 }
 
+@_silgen_name("_swift_tfc_PushDeviceScope")
+@usableFromInline @inline(never)
```

Comment:
Why does this need to be `@inline(never)`? Does deabstraction see calls to this function?

Reply:
I reverted methods like `_swift_tfc_PushDeviceScope` for now -- we will revisit them when/if we find the need to do compile-time device stack evaluation.